src/scene/light.js

```diff
@@ -328,111 +328,7 @@ class Light {
         this.key = key;
     }
 
-    // creates LUT texture used by area lights
```

Comment:
by removing these, perhaps there are some imports that can be removed from the start of this file?

Reply:
unused imports are removed (none are now flagged by lint)